README.md

```diff
@@ -49,7 +49,9 @@ Warning: Refusing to link: openssl
 followed by a compile error not being able to find one or more
 openssl/ include files, you may want to try:
 ```sh
-export DEP_OPENSSL_INCLUDE=/usr/local/include
+export DEP_OPENSSL_INCLUDE=`brew --prefix openssl`/include
```

Comment:
Please use
```
DEP_OPENSSL_INCLUDE="$(brew --prefix openssl)/include"
```
instead of backticks, per the [style guide](https://github.com/servo/servo/blob/master/docs/STYLE_GUIDE.md).